src/Shared/Extensions/PackageUrlExtension.cs

```diff
@@ -2,9 +2,12 @@
 
 namespace Microsoft.CST.OpenSource.Extensions;
 
+using Helpers;
 using System.IO;
 using System.Text.RegularExpressions;
 using PackageUrl;
+using System.Net;
```

Comment:
nit: Are both of these usings necessary?  I'm not sure as to what code might need them.